src/routes/plans/[id]/+page.svelte

```diff
@@ -699,7 +699,9 @@
           buttonText="Check Constraints"
           hasPermission={hasCheckConstraintsPermission}
           disabled={$simulationStatus !== Status.Complete}
-          statusBadgeText={($checkConstraintsStatus === Status.Complete || $checkConstraintsStatus === Status.Failed) &&
+          statusBadgeText={($constraintsStatus === Status.Complete ||
```

Comment:
Consider pulling this out into a reactive variable since there's a decent bit of logic in here? Also if a plan has no constraint runs and no associated constraints could we hide this status badge? Should be somewhat similar to scheduling goal runs?